posts/lte-examples.md

```diff
@@ -0,0 +1,692 @@
+      F.map (pullback.snd : pullback (π i) (π j) ⟶ X j).op (x j)),
+  -- there is a unique `s : F (op B)`
+    ∃! s : F (op B),
+  -- which restricts to `x i` over `X i` for all `i`.
+      ∀ i, F.map (π i).op s = x i
+    :=
+-- the proof...
+```
+The notation `C ⥤ D` is used in `mathlib` to denote the type of functors from `C` to `D` and `Cᵒᵖ` denotes the opposite category associated to `C`.
+In particular, the code above takes a presheaf `F` of abelian groups (at universe level `1`) on `Profinite.{0}`, and gives a necessary and sufficient condition for `F` to be a sheaf with respect to `proetale_topology`. 
+
+The category `Condensed.{0} Ab.{1}` of condensed abelian groups used in the statement of the main theorem is defined simply as the category of sheaves of abelian groups over `proetale_topology`.
+The category of sheaves over a Grothendieck topology `J` on a category `C`, taking values in `D` is usually denoted by `Sheaf J D`, but the two spellings for the category of condensed abelian groups are (definitionally!) the same.
+```lean
+example : Condensed.{0} Ab.{1} = Sheaf proetale_topology.{0} Ab.{1} := 
+rfl
+```
+
+The type of sheaves `Sheaf J D` (`J` and `D` as above) is again defined using dependent pairs, say `P`, where the first component of `P`, denoted above as `P.1`, is a presheaf on `C` with values in `D` and the second component `P.2` is a proof that `P.1` is a sheaf for `J`.
+We illustrate this in the case of condensed abelian groups with the following examples.
+```lean
+example (F : Condensed.{0} Ab.{1}) : Profinite.{0}ᵒᵖ ⥤ Ab.{1} := 
+F.1
+
+example (F : Condensed.{0} Ab.{1}) : 
+  presheaf.is_sheaf proetale_topology F.1 := 
+F.2
+
+example (F : Profinite.{0}ᵒᵖ ⥤ Ab.{1}) 
+  (hF : presheaf.is_sheaf proetale_topology F) :
+  Condensed.{0} Ab.{1} := 
+⟨F,hF⟩
+```
+Note that we again use Lean's anonymous constructor syntax `⟨F,hF⟩` in the last example.
+
+## A comment on universes
+One last comment about universes is warranted in this section.
+Just like `Profinite.{0}` is the category of profinite sets whose underlying type lives in `Type 0`, the category `Ab.{1}` is the category of abelian groups whose underlying type lives in `Type 1`.
+We need to bump the universe level of the category of abelian groups precisely because `Profinite.{0}` is a *large category*, meaning that `Profinite.{0} : Type 1`, while `X ⟶ Y : Type 0` for `X Y : Profinite.{0}`.
+Technically speaking, condensed mathematics in the sense of [Clausen-Scholze](https://www.math.uni-bonn.de/people/scholze/Condensed.pdf) works in ZFC by imposing cardinality bounds on profinite sets, whereas our approach more closely resembles that of *pyknotic objects*, in the sense of [Barwick-Haine](https://arxiv.org/abs/1904.09966).
+See the footnote on page 7 of [`Condensed.pdf`](https://www.math.uni-bonn.de/people/scholze/Condensed.pdf) and/or section 0.3 of [Barwick-Haine](https://arxiv.org/pdf/1904.09966.pdf) for the comparison between condensed and pyknotic sets.
```

Comment:
```suggestion
See the footnote on page 7 of [`Condensed.pdf`](https://www.math.uni-bonn.de/people/scholze/Condensed.pdf) and/or section 0.3 of [Barwick-Haine](https://arxiv.org/abs/1904.09966) for the comparison between condensed and pyknotic sets.
```
For consistency with the link above. (And because linking to `abs` is better in general, I think.)